roles/openshift_node_group/templates/node-config.yaml.j2

```diff
@@ -35,19 +46,12 @@ masterKubeConfig: node.kubeconfig
 networkConfig:
   mtu: {{ openshift_node_group_network_mtu }}
   networkPluginName: {{ openshift_node_group_network_plugin }}
-nodeIP: ""
-podManifestConfig: null
+networkPluginName: {{ openshift_node_group_network_plugin }}
 servingInfo:
   bindAddress: 0.0.0.0:10250
   bindNetwork: tcp4
-  certFile: server.crt
-  clientCA: node-client-ca.crt
-  keyFile: server.key
-  namedCertificates: null
+  clientCA: client-ca.crt
 volumeConfig:
   localQuota:
     perFSGroup: null
 volumeDirectory: {{ openshift_node_group_node_data_dir }}/openshift.local.volumes
-enable-controller-attach-detach:
-- 'true'
-networkPluginName: {{ openshift_node_group_network_plugin }}
```

Comment:
Ok, this is odd.  I can't see how this networkPluginName would be used at all...  Our config defines networkPluginName part of the NetworkConfig struct, not top-level in the node config.

Reply:
This is the legacy field.  It continues to be supported on the external struct and takes precedence over the inner.  We can probably simplify.